packages/blue-sdk/src/addresses.ts

```diff
@@ -102,6 +146,7 @@ export const addresses = {
     metaMorphoFactory: "0xFf62A7c278C62eD665133147129245053Bbf5918",
 
     wNative: "0x4200000000000000000000000000000000000006",
+    // Must implement USDC permit version 2 (otherwise breaks permit signatures).
```

Comment:
Might be an idea to expose a script that checks these requirements

Reply:
so that someone that extends the config could ensure it's compatible

Reply:
Better would be to have the version parameter in args of permit signature

Reply:
But we need the frontend to give it to us and if we push it too early they won't be compatible and it'll break signatures in production